CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Database Tools Changelog
 
+## 100.3.1
+
+_Released 2021-03-17_
+
+We are pleased to announce version 100.3.1 of the MongoDB Database Tools.
+
+This release includes various minor bug fixes.
```

Comment:
Can we call out TOOLS-2783 here specifically? Also, I think we should drop the word "minor".

Reply:
updated!